api/fastpm/cosmology.h

```diff
@@ -21,7 +27,7 @@ double Gamma_nu(FastPMCosmology * c);
 double Omega_ur(FastPMCosmology * c);
 double Omega_r(FastPMCosmology * c);
 
-double getFtable(int F_id, double y);
+double getFtable(int F_id, double y, FastPMCosmology * c);
```

Comment:
If you do that, getFtable would consume a FastPMFDInterp, and access cosmology parameters (if needed) from fdinterp->c.